merkle-tree/concurrent/tests/tests.rs

```diff
@@ -1069,3 +1153,63 @@ fn test_from_bytes_sha256_8_256_256_0() {
     const CANOPY: usize = 0;
     from_bytes::<Sha256, HEIGHT, CHANGELOG, ROOTS, CANOPY>()
 }
+
+#[test]
+fn test_changelog_event_v1() {
+    const HEIGHT: usize = 22;
```

Comment:
would testing time be bearable with height 26?

Reply:
Probably yes, let me try

Reply:
Yep, it works nicely.